### PR TITLE
add sample unit test for RestMLTrainingAction

### PR DESCRIPTION
**Pull request by ylwu-amzn**
Signed-off-by: Yaliang Wu <ylwu@amazon.com>

### Description
Add sample unit test for RestMLTrainingAction
@Zhangxunmt you can use similar way to add unit test for other rest actions
 

 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: plugin/src/test/java/org/opensearch/ml/rest/RestMLTrainingActionTests.java [hunk 1]
```diff
@@ -45,4 +112,47 @@ public void testRoutes() {
...
+        ArgumentCaptor<MLTrainingTaskRequest> argumentCaptor = ArgumentCaptor.forClass(MLTrainingTaskRequest.class);
+        verify(client, times(1)).execute(eq(MLTrainingTaskAction.INSTANCE), argumentCaptor.capture(), any());
+        MLInput mlInput = argumentCaptor.getValue().getMlInput();
+        verifyParsedMLInput(mlInput);
+    }
+
+    private RestRequest getRestRequest() {
+        Map<String, String> params = new HashMap<>();
+        params.put(PARAMETER_ALGORITHM, FunctionName.KMEANS.name());
+        final String requestContent = "{\"parameters\":{\"centroids\":3,\"iterations\":10,\"distance_type\":" +
```
Comment (Zhangxunmt) [inline plugin/src/test/java/org/opensearch/ml/rest/RestMLTrainingActionTests.java:137]: This is a good RestRequest sample.

**Review by Zhangxunmt — DISMISSED** [review]
lgtm

**Comment by Zhangxunmt** [comment]
LGTM, but needs to spotlessApply.

**Comment by ylwu-amzn** [comment]
> LGTM, but needs to spotlessApply.

Forgot to run spotlessApply, published a new commit

**Review by Zhangxunmt — APPROVED** [review]

**Pull request merged**